### DISCUSSION TITLE
Delete file option please 🙂✌️

### DISCUSSION THREAD

**Discussion by imadhava** [community/community#71119] [Mobile]
<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.135.0 (112648507)
Device: iPhone 11 (iOS 17.0.3)
</details>

**Comment by mattruggio** [comment]
Hey @I-Madhava thanks for the feature request! I can see our web experience offers this menu option:

![hidden-delete-options](https://github.com/community/community/assets/906763/6ca983d9-9fc3-41a4-b5be-f4e6b04d0ff7)

I went ahead and put in an internal feature request for this specific function. I cannot guarantee nor provide a timeline for this but we are always looking to increase feature parity with our web experience!

**Reply by imadhava** [reply]
Sir I was requesting delete file option for GitHub mobile! 

As we always need to switch between app and web to delete files!

**Reply by mattruggio** [reply]
> Sir I was requesting delete file option for GitHub mobile!

Sorry I was not clear enough, I was illustrating how the web experience has a delete file option compared to our mobile apps. The request was made for our mobile applications 🥂

**Reply by imadhava** [reply]
Hope it will be added soon 🙂✌️
